plugins/github/secrets.go

```diff
@@ -0,0 +1,92 @@
+//
+// Some entries are courtesy of git-secrets:
+// https://github.com/awslabs/git-secrets/blob/master/git-secrets#L233
+// Some entries are courtesy of gitleaks:
+// https://github.com/zricethezav/gitleaks/blob/f338bc584fbebcecb5dc372b40e2be86634f2143/config/gitleaks.toml
+// https://github.com/zricethezav/gitleaks/blob/f62617d7a6ddcb81ca72ee293a3d0c72bb738a67/examples/leaky-repo.toml
+///////////////////////////////////////////////////////////////////////////////
+var secretsChecks = []secretCheckInfo{
+	{"aws_access_key", "(A3T[A-Z0-9]|AKIA|AGPA|AIDA|AROA|AIPA|ANPA|ANVA|ASIA)[A-Z0-9]{16}", "key", "AWS"},
+	{"aws_secret_key", optQuote + aws + "(SECRET|secret|Secret)?_?(ACCESS|access|Access)?_?(KEY|key|Key)" + optQuote + connect + optQuote + "[A-Za-z0-9/\\+=]{40}" + optQuote, "key", "AWS"},
+	{"aws_mws_key", "amzn\\.mws\\.[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}", "key", "AWS"},
+	{"aws_account_id", optQuote + aws + "(ACCOUNT|account|Account)_?(ID|id|Id)?" + optQuote + connect + optQuote + "[0-9]{4}\\-?[0-9]{4}\\-?[0-9]{4}" + optQuote, "key", "AWS"},
+	{"facebook_secret_key", "(?i)(facebook|fb)(.{0,20})?(?-i)['\"][0-9a-f]{32}['\"]", "key", "Facebook"},
+	{"facebook_client_id", "(?i)(facebook|fb)(.{0,20})?['\"][0-9]{13,17}['\"]", "key", "Facebook"},
+	{"twitter_secret_key", "(?i)twitter(.{0,20})?['\"][0-9a-z]{35,44}['\"]", "key", "Twitter"},
+	{"twitter_client_id", "(?i)twitter(.{0,20})?['\"][0-9a-z]{18,25}['\"]", "client", "Twitter"},
+	{"github_personal_access_token", "ghp_[0-9a-zA-Z]{36}", "key", "Github"},
+	{"github_oauth_access_token", "gho_[0-9a-zA-Z]{36}", "key", "Github"},
+	{"github_app_token", "(ghu|ghs)_[0-9a-zA-Z]{36}", "key", "Github"},
+	{"github_refresh_token", "ghr_[0-9a-zA-Z]{76}", "key", "Github"},
+	{"linkedin_client_id", "(?i)linkedin(.{0,20})?(?-i)[0-9a-z]{12}", "client", "LinkedIn"},
+	{"linkedin_secret_key", "(?i)linkedin(.{0,20})?[0-9a-z]{16}", "secret", "LinkedIn"},
+	{"slack", "xox[baprs]-([0-9a-zA-Z]{10,48})?", "key", "Slack"},
+	{"asymmetric_private_key", "-----BEGIN ((EC|PGP|DSA|RSA|OPENSSH) )?PRIVATE KEY( BLOCK)?-----", "key", "AsymmetricPrivateKey"},
+	{"google_api_key", "AIza[0-9A-Za-z\\-_]{35}", "key", "Google"},
+	{"google_gcp_service_account", "\"type\": \"service_account\"", "key", "Google"},
+	{"heroku_api_key", "(?i)heroku(.{0,20})?[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}", "key", "Heroku"},
+	{"mailchimp_api_key", "(?i)(mailchimp|mc)(.{0,20})?[0-9a-f]{32}-us[0-9]{1,2}", "key", "Mailchimp"},
+	{"mailgun_api_key", "((?i)(mailgun|mg)(.{0,20})?)?key-[0-9a-z]{32}", "key", "Mailgun"},
+	{"paypal_braintree_access_token", "access_token\\$production\\$[0-9a-z]{16}\\$[0-9a-f]{32}", "key", "Paypal"},
+	{"picatic_api_key", "sk_live_[0-9a-z]{32}", "key", "Picatic"},
+	{"sendgrid_api_key", "SG\\.[\\w_]{16,32}\\.[\\w_]{16,64}", "key", "SendGrid"},
+	{"slack_webhook", "https://hooks.slack.com/services/T[a-zA-Z0-9_]{8}/B[a-zA-Z0-9_]{8,12}/[a-zA-Z0-9_]{24}", "key", "slack"},
```

Comment:
```suggestion
	{"slack_webhook", "^https://hooks\.slack\.com/services/T[a-zA-Z0-9_]{8}/B[a-zA-Z0-9_]{8,12}/[a-zA-Z0-9_]{24}$", "key", "slack"},
```
I think this would solve the code-scanning suggestions.